### DISCUSSION TITLE
Waveshare RS485 to POE ETH (B) poe doesn't power it

### DISCUSSION THREAD

**Discussion by darkrain-nl** [wills106/homeassistant-solax-modbus#216] [Q&A]
Hi,

Was wondering if anybody has a Waveshare RS485 to POE ETH (B) powered via POE?

I have such a device, but when connected to the switch nothing happens...

Kind regards,
darkrain

**Comment by wills106** [comment]
This might be a silly question, but I am guessing your switch actually provides POE (802.3af)?

**Comment by darkrain-nl** [comment]
Not a silly question, but yes it does, it is powering other devices just fine, capable of 802.3af and 802.3at.

It's just that I cannot find any information about how to test is with POE except the very limited info in the manual... So I am looking for some experience with this.

**Reply by wills106** [reply]
I wasn't aware of a POE version. I had a quick look at the manual and it mostly seems focused on the non POE.
You would assume it just works over POE like Camera's do.

Is it the white plastic looking body or the slim black metal one?

**Reply by darkrain-nl** [reply]
So I have already found the issue, although I had to open the unit to find it.

Turns out there is a little board inside that was not connected to anything anymore (I assume from transport)... So I connected that back to the only position it could go and now it works just fine.

**Reply by darkrain-nl** [reply]
It is the white plastic body btw...

**Reply by wills106** [reply]
Out of curiosity which Inverter are you connecting to?

**Reply by darkrain-nl** [reply]
That will be a Sofar solar, not sure about the exact type yet... But I will contribute to this project with information when it is connected and up and running.

**Reply by wills106** [reply]
Thanks, it's always good to find less used Inverters so we can improve the Integration.
(In comparison to how this started of as SolaX only focused Integration)

**Comment by J14111** [comment]
I have had this with two of these devices both brand new, you need to pop the lid off and push the PoE daughterboard in a bit further then it will work.
